server/views/certificate/data-vis.jade

```diff
@@ -1,5 +1,5 @@
 meta(name='viewport', content='width=device-width, initial-scale=1')
-link(rel='stylesheet', href='/bower_components/bootstrap/dist/css/bootstrap.min.css')
+link(rel='stylesheet', href='http://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css')
```

Comment:
AFAIK, webpack can also be configured to serve CSS from node_modules by defining a css loader in config. See [this discussion for a more context](https://github.com/webpack/webpack/issues/1789). For example, bootstrap css files can make it to the app via webpack and a `require()` in a JS file. Thus no need in CDNs.

Apologies if I'm speaking out of context, since I'm new to the codebase, would like to hear your opinion.